must-gather/gather

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source ${SCRIPT_DIR}/common
+BASE_COLLECTION_PATH="${1:-/must-gather}"
+export LOGFILE_PATH="${BASE_COLLECTION_PATH}/${LOGFILE_NAME}"
+
+mkdir -p "${BASE_COLLECTION_PATH}"
+cd ${BASE_COLLECTION_PATH}
+echo -e "must-gather logs are located at: '${LOGFILE_PATH}'"
+
+mkdir ${BASE_COLLECTION_PATH}/cache-dir
+export KUBECACHEDIR=${BASE_COLLECTION_PATH}/cache-dir
+
+print_usage() {
+    echo -e "Usage: /usr/bin/gather <gather-collection-path>"
+}
+
+case ${1} in
+  --help | -h )
+	print_usage
+	exit 1
+esac
```

Comment:
Could we move this to `main` so that all execution walk-through can start from `main` ?